### DISCUSSION TITLE
Please add More FPGA boards as presets

### DISCUSSION THREAD

**Discussion by Aggtur11** [logisim-evolution/logisim-evolution#1736] [Ideas]
I am working on projects based on Pynq-Z2 FPGA board. Could you please add this as well as a few other FPGA boards in the Software.

**Comment by maehne** [comment]
Our ressources are limited and we can only add support for boards, which we use ourselves. However, we are always open for PRs adding support to new boards. Have a look at this [message](https://github.com/logisim-evolution/logisim-evolution/issues/1427#issuecomment-1086597298) in issue #1427 to get started to add yourself for the FPGA board with Intel/Altera or AMD/Xilinx FPGAs.

**Comment by chuckb** [comment]
I did a video on adding a new board. You can find it here: https://youtu.be/9EfR1gAdsrA

**Reply by BFH-ktt1** [reply]
Hi @chuckb , I really like the video, thanks for the work. Just a small comment, you could have avoided putting all the inverters in by specifying for the segments and the common anodes "active low". Logisim will then automatically put in the inverters. As the basic idea in logisim is active-high behavior, this might make sense :-)

**Reply by chuckb** [reply]
Ah. Thanks for the tip!
